composer/callbacks/checkpoint_saver.py

```diff
@@ -78,6 +81,30 @@ def save_interval(state: State, event: Event):
     return save_interval
 
 
+def _extract_path_from_uri(uri: str):
```

Comment:
kind of the same question as below, but why is this different logic from the load path parsing?